rdmo/management/assets/js/actions/elementActions.js

```diff
@@ -342,14 +343,19 @@ export function fetchElementError(error) {
 
 // store element
 
-export function storeElement(elementType, element, back) {
+export function storeElement(elementType, element, back, elementAction=null) {
```

Comment:
Can we change that to `storeElement(elementType, element, elementAction, back)`? We just need to adjust the methods in the `edit` components. Looks cleaner.

Reply:
yes, I've changed it to that. 
Also updated this signature in the `dropElement` function and in other components..